app/views/projects/members/_create_modal.html.erb

```diff
@@ -9,7 +9,7 @@
         ) %>
       </p>
     </div>
-    <%= form_with(model: @new_member, url: namespace_project_members_path, method: :post) do |form| %>
+    <%= form_with(model: @new_member, url: namespace_project_members_path(**request.query_parameters), method: :post) do |form| %>
```

Comment:
**nitpick (blocking):** Same comment as in groups above

Reply:
Updated in  9d14e974555ff0bbe5c4b392f2e0b47b8a2b1865